ExtendAdminConsoleTest#testAdminUiTodoApp fails with NoSuchElementException

### Describe the bug

When running the automated integration tests, the ExtendAdminConsoleTest.testAdminUiTodoApp test fails with a NoSuchElementException. The "add" or empty action button can't be located using the XPath //button[@data-testid='there-are-no-items-empty-action'] inside the ExtendedAdminPage.clickAddButton method.

### Version

Nightly 

### Expected behavior

The test should successfully find the empty state action button, click it, and proceed with the Admin UI Todo application test workflow without throwing an exception.

### Actual behavior

```
[ERROR]   ExtendAdminConsoleTest.testAdminUiTodoApp:99 » NoSuchElement no such element: Unable to locate element: {"method":"xpath","selector":"//button[@data-testid='there-are-no-items-empty-action']"} 
```

### How to Reproduce?

- Open the repository on a machine running macOS
- Run the ExtendAdminConsoleTest class using:  mvn verify -Dit.test=ExtendAdminConsoleTest -Pextension -rf :extend-admin-ui  


### Anything else?

_No response_

--- a/extension/extend-admin-console-spi/src/test/java/org/keycloak/quickstart/ExtendedAdminPage.java
+++ b/extension/extend-admin-console-spi/src/test/java/org/keycloak/quickstart/ExtendedAdminPage.java
@@ -34,7 +34,7 @@ public class ExtendedAdminPage {
     private WebElement todoMenuItem;
 
     @FindBy(
-            xpath = "//button[@data-testid='there-are-no-items-empty-action']"
+            xpath = "//*[text()='Create item' or @data-testid='there-are-no-items-empty-action']"
     )
     private WebElement addButton;
 

